conf/locale/locale_en-US.ini

```diff
@@ -587,11 +587,16 @@ settings.danger_zone = Danger Zone
 settings.transfer = Transfer Ownership
 settings.transfer_desc = Transfer this repository to another user or to an organization in which you have admin rights.
 settings.new_owner_has_same_repo = The new owner already has a repository with same name. Please choose another name.
+settings.convert = Convert To Regular Repository
+settings.convert_desc = You can convert this mirror to a regular repository. This cannot be reversed.
 settings.delete = Delete This Repository
 settings.delete_desc = Once you delete a repository, there is no going back. Please be certain.
 settings.transfer_notices_1 = - You will lose access if new owner is a individual user.
 settings.transfer_notices_2 = - You will conserve access if new owner is an organization and if you're one of the owners.
 settings.transfer_form_title = Please enter following information to confirm your operation:
+settings.convert_notices_1 = - This operation will convert this repository mirror into a regular repository and cannot be undone.
```

Comment:
I refactored the file a bit and moved all "transfered", "delete", and "convert" strings together. They were all scattered all over the place. Will be pushed with the next commit.